### ISSUE TITLE
Error when adding existing fields to Stencil

### ISSUE THREAD

**Issue by cdccairns** [verbb/formie#1100]
### Describe the bug

Formie returns the following error when adding existing fields to a Stencil:

```
verbb\formie\services\Forms::getFormById(): Argument #1 ($id) must be of type int, null given, called in /vendor/verbb/formie/src/controllers/FormsController.php on line 402
```

### Steps to reproduce

1. Add new Stencil via Formie → Settings → Stencils
2. Add fields to Stencil
3. Add (another) new Stencil
4. Click _Add existing fields_

### Form settings

N/A

### Craft CMS version

4.2.5.1

### Plugin version

2.0.12

### Multi-site?

Yes

### Additional context

```
{
  "trace": [
    {
      "file": "/vendor/verbb/formie/src/controllers/FormsController.php",
      "line": 402,
      "function": "getFormById",
      "class": "verbb\\formie\\services\\Forms",
      "type": "->"
    },
    {
      "function": "actionGetExistingFields",
      "class": "verbb\\formie\\controllers\\FormsController",
      "type": "->"
    },
    {
      "file": "/vendor/yiisoft/yii2/base/InlineAction.php",
      "line": 57,
      "function": "call_user_func_array"
    },
    {
      "file": "/vendor/yiisoft/yii2/base/Controller.php",
      "line": 178,
      "function": "runWithParams",
      "class": "yii\\base\\InlineAction",
      "type": "->"
    },
    {
      "file": "/vendor/yiisoft/yii2/base/Module.php",
      "line": 552,
      "function": "runAction",
      "class": "yii\\base\\Controller",
      "type": "->"
    },
    {
      "file": "/vendor/craftcms/cms/src/web/Application.php",
      "line": 301,
      "function": "runAction",
      "class": "yii\\base\\Module",
      "type": "->"
    },
    {
      "file": "/vendor/craftcms/cms/src/web/Application.php",
      "line": 625,
      "function": "runAction",
      "class": "craft\\web\\Application",
      "type": "->"
    },
    {
      "file": "/vendor/craftcms/cms/src/web/Application.php",
      "line": 280,
      "function": "_processActionRequest",
      "class": "craft\\web\\Application",
      "type": "->"
    },
    {
      "file": "/vendor/yiisoft/yii2/base/Application.php",
      "line": 384,
      "function": "handleRequest",
      "class": "craft\\web\\Application",
      "type": "->"
    },
    {
      "file": "/www/index.php",
      "line": 12,
      "function": "run",
      "class": "yii\\base\\Application",
      "type": "->"
    }
  ]
}
```

**Comment by engram-design** [comment]
You actually shouldn't be able to select an existing field when editing a stencil, sorry.

Should be fixed for the next version. To get this early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-4 as 2.0.12",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [2.0.13](https://github.com/verbb/formie/releases/tag/2.0.13)
